aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -63,7 +63,7 @@ Update the `SeedData` class so that it provides a value for the new column. A sa
 [!code-csharp[Main](./start-mvc/sample2/src/MvcMovie/Models/SeedDataRating.cs?highlight=6&range=25-32)]
 
 >[!WARNING]
-> You must stop IIS Express before you run the `dotnet ef` commands.   See [To Stop IIS Express:](adding-model.md#stop-iis-express-reference-label)
+> You must stop IIS Express before you run the `dotnet ef` commands. In Visual Studio 2017 RC this step isn't necessary, but you should close the browser window and stop debugging. See [To Stop IIS Express:](adding-model.md#stop-iis-express-reference-label)
```

Comment:
Yes, no longer necessary. It was required in the very last RTM-candidate that I used, but RTM fixed the problem.